data/ecosystems/r/ronin.toml

```diff
@@ -1,220 +1,58 @@
 # Ecosystem Level Information
 title = "Ronin"
 
-sub_ecosystems = ["Moralis"]
+sub_ecosystems = [
+  "Axie Infinity",
+  "Moralis",
+  "Footprint Analytics",
+  "Impossible Finance",
+  "Sablier Finance",
+  "RAMP",
+  "APY Vision",
+  "Clique",
+  "Pyth Network",
+  "Nansen",
+  "Binance Exchange",
+  "Coinbase",
+  "DappRadar"
```

Comment:
I was using ethereum.toml as reference, which also had Binance Exchange and DappRadar listed. 
DappRadar is collaborating with Sky Mavis since 2018 and even maintains an active [governing validator for Ronin](https://app.roninchain.com/validator/0x8eec4f1c0878f73e8e09c1be78ac1465cc16544d ).

Anyway I removed all of these entries for now with d1e6e20